io/net/virt-net/network_virtualization.py

```diff
@@ -801,3 +805,7 @@ def tearDown(self):
         self.session_hmc.quit()
         if 'vios' in str(self.name.name):
             self.session.quit()
+        cmd = "echo 'module ibmvnic -p; func send_subcrq -p' > /sys/kernel/debug/dynamic_debug/control"
+        result = process.run(cmd, shell=True, ignore_status=True)
+        if result.exit_status:
+            self.fail("failed to disable debug mode")
```

Comment:
@smruti77 Is this working ?

Reply:
@narasimhan-v  this will come as Error.

Reply:
So let us not fail here. Just make it self.log.debug

Reply:
ok